packages/backend/src/graphql/schema.graphql

```diff
@@ -747,6 +747,8 @@ type Peer implements Model {
   liquidity: UInt64
   "The date and time when the peer was created."
   createdAt: String!
+  "Unique identifier of the tenant associated with the peer."
+  tenantId: ID!
```

Comment:
I think `tenantId` still needs to be added to the `CreatePeerInput`, so that the `CreatePeer` resolver can leverage the `ForTenantId` context.